example/src/main/java/io/netty/example/http2/tiles/Html.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.example.http2.tiles;
+
+import io.netty.example.http2.tiles.http.HttpServer;
+import io.netty.example.http2.tiles.http2.Http2Server;
+
+import java.util.Random;
+
+/**
+ * Static and dynamically generated HTML for the example.
+ */
+public class Html {
+
+  public static final String IP = System.getProperty("ip", "127.0.0.1");
+
+  public static final byte[] FOOTER = "</body></html>".getBytes();
+
+  public static final byte[] HEADER = ("<html><head lang=\"en\"><title>Netty HTTP/2 Example</title>"
```

Comment:
Could we make this a proper HTML5? e.g. `<!DOCTYPE html>`, `<meta charset="UTF-8">`, ...